base/float.jl

```diff
@@ -414,9 +422,121 @@ muladd(x::T, y::T, z::T) where {T<:IEEEFloat} = muladd_float(x, y, z)
 # TODO: faster floating point fld?
 # TODO: faster floating point mod?
 
-rem(x::T, y::T) where {T<:IEEEFloat} = rem_float(x, y)
+function unbiased_exponent(x::T) where {T<:IEEEFloat}
+    exp = (reinterpret(Unsigned, x) & exponent_mask(T)) >> mantissa_width(T)
+    return exp
+end
+
+function explicit_mantissa_noinfnan(x::T) where {T<:IEEEFloat}
+    subnormal = !issubnormal(x) ? significand_mask(T) + uinttype(T)(1) : uinttype(T)(0)
+    return subnormal | (significand_mask(T) & reinterpret(Unsigned, x))
+end
+
+function make_value(number::T, ep) where {T<:Unsigned}
```

Comment:
Is there a more descriptive name for the function than "make value"? lol

Also, very pedantic comment, but the majority of the functions here are taking `T` to be an `IEEEFloat`. If it were me, I'd probably make this `U`, `T = floattype(U)`, and `S = signed(U)`. But of course that's just, like, my opinion, man.

Reply:
That was the name in the libc implementation, I'm open to any and all bike shedding. Also your opinion is valuable ;)

Reply:
I do!